### PR TITLE
Latest recommendations and stories data refactor.

### PR DESCRIPTION
**Pull request by itsyogesh**

### CODE REVIEW

**Comment by itsyogesh** [comment]
image

File: src/components/Sidebar/LatestRecommendations.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import { FormattedRelative } from 'react-intl';
+import { Link } from 'react-router-dom';
+import BodyShort from '../Story/BodyShort';
+import './LatestRecommendations.less';
+
+const renderRecommendedPost = ({ title, author, created }) => (
+  <div className="Recommendation">
+    <div className="Recommendation__text">
+      <div className="Recommendation__title">
+        <BodyShort body={title} length={80} />
```
Comment (bonustrack) [inline src/components/Sidebar/LatestRecommendations.js]: Title must link to the related post with this format `/:parent_permlink/@:author/:permlink` . (but stay with grey color)

**Comment by bonustrack** [comment]
There should be a link in bottom of the box with a link "See All Recommendations". Similar than "Discover More People" link in "Intersting People" component: 
image

**Comment by bonustrack** [comment]
And also in the same way we need a link "See All Comments" on "Latest comments" component.

**Pull request merged**